koku/masu/management/commands/restart_celery.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# pragma: no cover
+"""Command to start celery worker with autoreload."""
+import os
+import shlex
+import signal
+import subprocess
+
+import psutil
+from django.core.management.base import BaseCommand
+from django.utils import autoreload
+
+
+def get_pid(name):
+    return [proc.pid for proc in psutil.process_iter() if name in proc.name()]
+
+
+def restart_celery():
+    for pid in get_pid("celery"):
+        os.kill(pid, signal.SIGTERM)
+    cmd = "celery -A koku worker --without-gossip -P solo -l info -Q celery,download,hcs,ocp,priority,summary,cost_model,refresh,subs_extraction,subs_transmission"  # noqa: E501
+    subprocess.run(shlex.split(cmd))
```

Comment:
Rather than using `shlex`, this could be a list. It's a bit easier to read and maintain that way.
```suggestion
    cmd =  [
        "celery"
        "-A", "koku", "worker",
        "--without-gossip",
        "-P", "solo",
        "-l", "info",
        "-Q", "celery,download,hcs,ocp,priority,summary,cost_model,refresh,subs_extraction,subs_transmission",
    ]
    subprocess.run(cmd)
```